### PR TITLE
feat(charts): import charts from @patternfly/react-charts/victory

### PR DESCRIPTION
**Pull request by dlabrecq**
This code mod updates existing import paths to our victory directory.

https://github.com/patternfly/patternfly-react/issues/11090

### CODE REVIEW

**Review by dlabaj — APPROVED** [review]
LGTM.  Thanks.

File: packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### chartsVictory-promoted [(#11091)](https://github.com/patternfly/patternfly-react/pull/11091)
```
Comment (wise-king-sullyman) [inline packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.md:1]: We should update the rule name here.

File: packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.test.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      `import {\n\t${specifier}\n} from '@patternfly/react-charts/victory';`,
+      [{ message: errorMessage, type: "ImportDeclaration" }]
+    )
+  );
+  invalidTests.push(
+    createInvalidTest(
+      `import { ${specifier} } from '@patternfly/react-charts/dist/dynamic/components/index.js';`,
+      `import {\n\t${specifier}\n} from '@patternfly/react-charts/victory';`,
+      [{ message: errorMessage, type: "ImportDeclaration" }]
+    )
```
Comment (wise-king-sullyman) [inline packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.test.ts:99]: Is it intended that for the cjs and dynamic imports it changes it to just `/victory`, but it keeps the esm import as explicitly esm?

File: packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImportMovedOutput.tsx [hunk 3]
```diff
@@ -0,0 +1 @@
+import { Chart } from '@patternfly/react-charts/victory';
```
Comment (wise-king-sullyman) [inline packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImportMovedOutput.tsx:1]: Nit: for the output files we like to have them as the 1:1 output of running the fixer, which here would have Chart on a new line.

The reasoning for this is that in the near future we will have a CI check that makes sure the output matches the input after autofixes are applied.

**Review by wise-king-sullyman — COMMENTED** [review]
Thanks for putting this codemod together! Few small things:

File: packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.md (hunk 1 above)
Comment (dlabrecq) [reply packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.md:1]: Yes, this should match. I renamed the files and didn't notice the comment

File: packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImportMovedOutput.tsx (hunk 3 above)
Comment (dlabrecq) [reply packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImportMovedOutput.tsx:1]: I don't really understand why Chart would be on a new line. I had to fix all that for Cost Management's linter.

File: packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.test.ts (hunk 2 above)
Comment (dlabrecq) [reply packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.test.ts:99]: I copied the Modal tests here. I'll update it

**Review by wise-king-sullyman — APPROVED** [review]

**Pull request merged**